frontend/test/jest/utils.tsx

```diff
@@ -115,3 +118,14 @@ export const getNonDiscountedProduct = (originalPrice: number = 49.99) => {
    });
    return product;
 };
+
+export const getProductOfType = (type: 'battery' | 'tool' | 'part') => {
```

Comment:
Are we using this function anywhere or are we planning to replace it in a later pull?

Reply:
I was thinking of using this function in `ProductSection.test.tsx` but it looks like we don't think need to. 👍